<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>750</width>
    <height>480</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Cardio-monitor</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <layout class="QHBoxLayout" name="upperToolbox">
      <item>
       <widget class="QPushButton" name="startButton">
        <property name="text">
         <string>Start</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pauseButton">
        <property name="text">
         <string>Pause</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="stopButton">
        <property name="text">
         <string>Stop</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="saveButton">
        <property name="text">
         <string>Save</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QGridLayout" name="gridLayout">
      <item row="1" column="0">
       <widget class="QPushButton" name="rescanButton">
        <property name="text">
         <string>Rescan</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QComboBox" name="serialportList"/>
      </item>
      <item row="1" column="2">
       <widget class="QGroupBox" name="groupBox">
        <property name="title">
         <string>Display Mode</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <widget class="QRadioButton" name="displayUnits">
           <property name="text">
            <string>Units</string>
           </property>
           <attribute name="buttonGroup">
            <string notr="true">displayMode</string>
           </attribute>
          </widget>
         </item>
         <item>
          <widget class="QRadioButton" name="displayVolts">
           <property name="text">
            <string>Volts</string>
           </property>
           <attribute name="buttonGroup">
            <string notr="true">displayMode</string>
           </attribute>
          </widget>
         </item>
        </layout>
        <zorder>displayVolts</zorder>
        <zorder>displayUnits</zorder>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="CardioPlot" name="cardioPlot" native="true">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" name="timeRangeLayout">
      <item>
       <widget class="QLabel" name="label">
        <property name="text">
         <string>Time Range</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QSlider" name="timeSlider">
        <property name="minimum">
         <number>1</number>
        </property>
        <property name="maximum">
         <number>5</number>
        </property>
        <property name="pageStep">
         <number>1</number>
        </property>
        <property name="value">
         <number>5</number>
        </property>
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="tickPosition">
         <enum>QSlider::TicksBelow</enum>
        </property>
        <property name="tickInterval">
         <number>1</number>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="valueRangeLayout">
      <item>
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>Value Range</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>From</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QSpinBox" name="fromSpin"/>
      </item>
      <item>
       <widget class="QLabel" name="label_4">
        <property name="text">
         <string>To</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QSpinBox" name="toSpin"/>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>CardioPlot</class>
   <extends>QWidget</extends>
   <header location="global">src/cardioplot.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
 <buttongroups>
  <buttongroup name="displayMode"/>
 </buttongroups>
</ui>
